lib/mix/tasks/phx.gen.schema.ex

```diff
@@ -86,6 +86,26 @@ defmodule Mix.Tasks.Phx.Gen.Schema do
   Generated migration can use `binary_id` for schema's primary key
   and its references with option `--binary-id`.
 
+  ## prefix
+
+  By default migrations and schemas are generated without a prefix.
+
+  For PostgreSQL this sets the "SCHEMA" (typically set via `search_path`)
+  and for MySQL it sets the database for the generated migration and schema.
+
+  The prefix can be used to thematically organize your tables on the database level.
+
+  > **Warning**
+  >
+  > The flag does not generate migrations to create
+  > the schema / database. This needs to be done manually or in a
+  > separate migration.
+  {:.warning}
```

Comment:
Ah, you are putting the warning in the wrong place, it should be with the title, which explains why the markup is not proper. :)

Reply:
Ah okay. I wasn't sure about the correct syntax, so I took it from an old GitHub issue and it seemed to work 😄